.ci/update-beats.yml

```diff
@@ -64,5 +63,4 @@ targets:
       command: .ci/scripts/update-beats.sh
       environments:
         - name: PATH
-        - name: GOPATH
```

Comment:
chore, it's not required and it iwll help with removing the error/warning in the console